.storybook/preview-head.html

```diff
@@ -4,6 +4,7 @@
   body {
     margin: 0;
     padding: 0;
+    font-size: 13px;
```

Comment:
rem inherits from the root elements from size, so 1rem === 13px.

Reply:
Does it mean we need to define the root font size in each app?

Reply:
Yes, you need to set the font-size on the `html` root element.

Reply:
So if someone wanted to use `ui-kit` for a static site or a documentation website, they could set the root font-size to something more appropriate, like 16px.

Reply:
That's the cool part though :)

Reply:
Yes of course, just wanted to make sure 😄

I guess we need to document it and make it clear. What is the default font size of the browser btw?

Reply:
[This](https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984) is a decent article that explains it well.

Generally the default is 16px

Reply:
FYI we are already setting the font size to the 13px for each app.

[ref](https://github.com/commercetools/merchant-center-application-kit/blob/master/packages/application-shell/src/components/application-shell/reset.mod.css)

Reply:
I still think this would be a breaking change, right? I mean, to be frank, nobody uses UIKit apart from us but I guess we want to do semver.

Reply:
Yep, I would say breaking. 

Interesting [articlee](https://medium.com/eightshapes-llc/visual-breaking-change-in-design-systems-1e9109fac9c4) on this

Reply:
![7a93fec5-e115-4cda-b6c3-6bae147cb5da](https://user-images.githubusercontent.com/1110551/51790286-4f8c1380-2193-11e9-8c45-c73954fc021e.jpeg)

Reply:
I agree and see the point. However, I think we just enable supporting REMs here which is cool. Whatever we then do on documentation sites and in apps is another thing? I would also argue, even though I agree with the point the article makes, that there may be a difference between web app and website. I know the lines are incredibly blurry these days but I think an app (e.g the MC) has to decide how much it can support variable (browser set) font sizes. In a perfect world anything should just let a browser and user decide. Totally agreed. Thanks for sharing. Really interesting.

Reply:
Yeah I agree, I also think for apps is a bit different.

I'm totally fine btw for going with `rem` for ui-kit, in fact I even proposed that some time ago 😜 https://github.com/commercetools/ui-kit/pull/387#discussion_r245276573

Reply:
@emmenko yeah I remember, that's what gave me the idea to make this PR 😆